src/transport/Session.h

```diff
@@ -78,6 +78,10 @@ class Session
     virtual System::Clock::Timestamp GetMRPBaseTimeout()               = 0;
     virtual System::Clock::Milliseconds32 GetAckTimeout() const        = 0;
 
+    // Returns a suggested timeout value based on the session type and the given upper layer processing timeout.
```

Comment:
_"Returns a suggested timeout value based on the round-trip time it takes for the peer at the other end of the session to receive a message, process it and send it back. This is computed based on the session type, the type of transport, sleepy characteristics of the target and a caller-provided value for the time it takes to process a message at the upper layer on the target"._